File: 0633-sum-of-square-numbers/0633-sum-of-square-numbers.java
import java.util.*;
class Solution {
    public boolean judgeSquareSum(int c) {
       int t=(int)Math.floor(Math.sqrt(c));
       for(int i=0;i<=t;i++){
        if((Math.sqrt(c-i*i))==Math.floor(Math.sqrt(c-i*i))){
            return true;
        }
       }
       return false;
    }
}

File: 0633-sum-of-square-numbers/0633-sum-of-square-numbers.py
import math
class Solution:
    def judgeSquareSum(self, c: int) -> bool:
        t=math.floor(math.sqrt(c))
        if c==2 or c==1:
            return True
        for i in range(t+1):
            if (math.sqrt(c-(i**2))).is_integer():
                print(math.sqrt(c-i**2))
                print(i)
                return True
            
        return False